lib/Driver/ToolChains.cpp

```diff
@@ -1147,9 +1169,32 @@ toolchains::GenericUnix::constructInvocation(const LinkJobAction &job,
   }
   llvm::sys::path::append(RuntimeLibPath,
                           getPlatformNameForTriple(getTriple()));
+
+ // On Linux and FreeBSD (really, ELF binaries) we need to add objects
+ // to provide markers and size for the conformance sections.
+ // This wouldn't be necessary if the link was done by  the swift binary.
```

Comment:
This comment isn't relevant (nor is the Radar reference) because this *is* linking with the Swift binary.